src/jcadplugin/plugins.ts

```diff
@@ -16,12 +16,15 @@ const activate = (
   tracker: WidgetTracker<IJupyterCadWidget>,
   themeManager: IThemeManager
 ): void => {
+  const { user } = app.serviceManager;
```

Comment:
IIRC the `_user` property of `AnnotationModel` is also the current user, can we use this property instead of getting it from here?

Reply:
Thanks I'll have a look